### ISSUE TITLE
Automatic change to dark theme.

### ISSUE THREAD

**Issue by fury21** [richardtop/CalendarKit#232]
### New Issue Checklist

- [x] Use the latest version of CalendarKit
- [x] I've read the [Contribution Guidelines](https://github.com/richardtop/CalendarKit/blob/master/CONTRIBUTING.md)
- [x] I've searched for [existing GitHub issues](https://github.com/richardtop/CalendarKit/issues)

### Issue Description

Hi! Propose a to remove button, "dark", and change themes on dark and back to do automatic in dependence from themes chosen by on phone.

   override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
        super.traitCollectionDidChange(previousTraitCollection)
              
       changeStyle()
    }
    
    private func changeStyle() {
        var style: CalendarStyle!
        
        if traitCollection.userInterfaceStyle == .light || traitCollection.userInterfaceStyle == .unspecified {
            style = StyleGenerator.defaultStyle()
        } else if traitCollection.userInterfaceStyle == .dark {
            style = StyleGenerator.darkStyle()
        } else {
            style = StyleGenerator.defaultStyle()
        }
        
        updateStyle(style)
        
        navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor:style.header.swipeLabel.textColor]
        reloadData()
    }

=========

And also add changeStyle() to viewDidLoad()

##### Code I'm using with CalendarKit

```Swift

[INSERT CODE HERE]

```

#### Result I am trying to achieve

[INSERT API OR UI MOCKUPS HERE]

**Comment by richardtop** [comment]
The dark style attached is just for example purposes. The user of the library has to either copy that style or implement a custom one in order to support the dark mode.

However, it’s a good suggestion to incorporate such style into the library source code and not just an example.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fury21** [comment]
An example of a dark theme turned out very well and it should be made the main dark theme

**Comment by richardtop** [comment]
Ok, great. I'll incorporate it into the project.

**Comment by jbarros35** [comment]
on the iOS 13 you have to comply with new styles to work.
I'm sending you how I'm using this. Basically you define your own set of colors and after that iOS 13 do the magic for you.

```
self.dayView = DayView(calendar: calendar)
        view = dayView
        
        let timeline = TimelineStyle()
        timeline.eventsWillOverlap = false
        timeline.verticalDiff = 160
        timeline.lineColor = UIColor(named: "onSecondary")!
        timeline.timeColor = UIColor(named: "onSecondary")!
        timeline.backgroundColor = UIColor(named: "background")!
        timeline.allDayStyle.backgroundColor = UIColor(named: "onSecondary")!
        timeline.allDayStyle.allDayColor = UIColor(named: "secondary")!
        
        let style = CalendarStyle()
        style.timeline = timeline
        
        let daySymbols = DaySymbolsStyle()
        daySymbols.weekDayColor = UIColor(named: "onBackground")!
        daySymbols.weekendColor = UIColor(named: "onSecondary")!
        
        let swipeLabel = SwipeLabelStyle()
        swipeLabel.textColor = UIColor(named: "background")!
        
        let selector = DaySelectorStyle()
        selector.activeTextColor = UIColor(named: "onBackground")!
        selector.inactiveTextColor = UIColor(named: "onSecondary")!
        selector.selectedBackgroundColor = UIColor(named: "background")!
        
        let header = DayHeaderStyle()
        header.daySelector = selector
        header.daySymbols = daySymbols
        header.swipeLabel = swipeLabel
        header.backgroundColor =  UIColor(named: "background")!
        style.header = header
        self.dayView.updateStyle(style)
```

**Comment by richardtop** [comment]
Available in 0.13.8
